package.json

```diff
@@ -35,6 +35,7 @@
     "morgan": "^1.9.1",
     "object-hash": "^1.3.1",
     "oidc-client": "^1.7.0",
+    "path-match": "^1.2.4",
```

Comment:
Is there not a path match with `react-router` already? like https://knowbody.github.io/react-router-docs/api/match.html